packages/@sanity/cli/test/exportImport.test.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'path'
+import {stat} from 'fs/promises'
+import {describeCliTest, testConcurrent} from './shared/describe'
+import {
+  getTestRunArgs,
+  runSanityCmdCommand,
+  studiosPath,
+  studioVersions,
+} from './shared/environment'
+
+describeCliTest('CLI: `sanity dataset export` / `import`', () => {
+  describe.each(studioVersions)('%s', (version) => {
+    const testRunArgs = getTestRunArgs(version)
+
+    testConcurrent('export', async () => {
+      const result = await runSanityCmdCommand(version, [
+        'dataset',
+        'export',
+        'production',
+        testRunArgs.exportTarball,
+        '--overwrite',
+      ])
+      expect(result.stdout).toMatch(/export finished/i)
+      expect(result.code).toBe(0)
+
+      const stats = await stat(path.join(studiosPath, version, testRunArgs.exportTarball))
+      expect(stats.isFile()).toBe(true)
+      expect(stats.size).toBeGreaterThanOrEqual(5000)
```

Comment:
What is `5000` here? Is this a threshold we can assume is accurate? Would it add clarity to give a it name, e.g. `const TARBALL_MIN_TYPICAL_SIZE = 5000` with comment?

Reply:
Just me being lazy. I've replaced it with a proper tarball contents check (for an ndjson file + asset) - the functionality itself is provided by `@sanity/export`, so will leave the "proper" tests for that module.